### DISCUSSION TITLE
Lightspeed

### DISCUSSION THREAD

**Discussion by CrazyDibber** [3kh0/ext-remover#454] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there a way to use point blank on the lightspeed filter agent?

**Comment by Aka-but-nice** [comment]
You will have to use old point-blank, which is for all extensions, but very laggy

**Comment by Crazygird** [comment]
bruh

**Comment by Crazygird** [comment]
WAIT!!! I FOUND A WAY!!! If you click licences and click open in new window, and spam esc it opens untitled. I tried javascript:alert("test")  and it worked!!!

**Comment by Crazygird** [comment, marked as answer]
this thread is over
